src/components/shared/MarkerTooltipContents.js

```diff
@@ -53,41 +53,72 @@ function getMarkerDetails(data: MarkerPayload): React$Element<*> | null {
       }
       case 'GCMinor': {
         if (data.nursery) {
-          return (
-            <div className="tooltipDetails">
-              {_markerDetail('gcreason', 'Reason', data.nursery.reason)}
-            </div>
-          );
+          const nursery = data.nursery;
+          switch (nursery.status) {
+            case 'complete':
+              return (
+                <div className="tooltipDetails">
+                  {_markerDetail('gcreason', 'Reason', nursery.reason)}
+                </div>
+              );
+            case 'nursery disabled':
+              return (
+                <div className="tooltipDetails">
+                  {_markerDetail('gcstatus', 'Status', 'Nursery disabled')}
+                </div>
+              );
+            case 'nursery empty':
+              return (
+                <div className="tooltipDetails">
+                  {_markerDetail('gcstatus', 'Status', 'Nursery empty')}
+                </div>
+              );
+            default:
+              return null;
+          }
         } else {
           return null;
         }
       }
       case 'GCMajor': {
-        let zones_collected_total;
-        if (data.timings.zones_collected && data.timings.total_zones) {
-          zones_collected_total =
-            data.timings.zones_collected + ' / ' + data.timings.total_zones;
+        const timings = data.timings;
+        switch (timings.status) {
+          case 'aborted':
+            return (
+              <div className="tooltipDetails">
+                {_markerDetail('status', 'Status', 'Aborted (OOM)')}
+              </div>
+            );
+          case 'completed': {
+            let zones_collected_total;
+            if (timings.zones_collected && timings.total_zones) {
```

Comment:
Honnestly I don't remember, but it probably wasn't because they might be zero.  I'd still want to report that.  I've found some instances where they are zero and I'm not sure what the GC is doing in that case.  Something else for me to investigate ;-)